doc/source/whatsnew/v1.5.0.rst

```diff
@@ -244,7 +244,7 @@ Conversion
 - Bug in :meth:`Float64Index.astype` to unsigned integer dtype incorrectly casting to ``np.int64`` dtype (:issue:`45309`)
 - Bug in :meth:`Series.astype` and :meth:`DataFrame.astype` from floating dtype to unsigned integer dtype failing to raise in the presence of negative values (:issue:`45151`)
 - Bug in :func:`array` with ``FloatingDtype`` and values containing float-castable strings incorrectly raising (:issue:`45424`)
--
+- Bug in :meth:`objects_to_datetime64ns` throws OverflowError when converting string to datetime64ns. The issue was observed when comparing string and datetime64ns objects. (:issue:`45506`)
```

Comment:
don't use internal methods here. just say something about comparing strings and datetimes in a Series.

Reply:
Done